Formula/overcommit.rb

```diff
@@ -0,0 +1,31 @@
+class Overcommit < Formula
+  desc "Utility to install, configure, and extend Git hooks"
+  homepage "https://github.com/sds/overcommit"
+  url "https://github.com/sds/overcommit/archive/v0.49.1.tar.gz"
+  sha256 "6bf60311211efdd036abe14d62cd314226f99d6c0676810e79df555a72bc9ac3"
+
+  depends_on "ruby"
```

Comment:
What version does it need?

Reply:
2.4 & above: https://github.com/sds/overcommit/blob/master/.travis.yml#L6